src/test/kotlin/no/nb/bikube/newspaper/service/NewspaperServiceTest.kt

```diff
@@ -140,6 +149,7 @@ class NewspaperServiceTest {
     fun `createTitle should return Title object with default values from Title with only name and materialType`() {
         every { collectionsRepository.createTextsRecord(any()) } returns Mono.just(collectionsModelMockTitleC)
         every { collectionsRepository.getSingleCollectionsModelWithoutChildren(any()) } returns Mono.just(collectionsModelMockTitleC)
+        every { uniqueIdService.getUniqueId() } returns "1600000000"
```

Comment:
Ser ut som denne og de like heller kunne vært i en beforeEach, så slipper man å skrive inn i alle testene

Reply:
Done and done